test/testenv/charmuseum.go

```diff
@@ -66,5 +73,17 @@ func DeployChartMuseum(ctx context.Context, input DeployChartMuseumInput) {
 		DeploymentName:       "chartmuseum",
 		ServiceName:          "chartmuseum-service",
 		PortName:             "http",
+		CustomIngressConfig:  input.CustomIngressConfig,
+		Variables:            input.Variables,
 	})
 }
+
+// PreChartMuseumInstallHook is a pre-install hook for ChartMuseum.
+func PreChartMuseumInstallHook(chartMuseumInput *DeployChartMuseumInput, e2eConfig *clusterctl.E2EConfig) {
+	infrastructureType := e2e.ManagementClusterEnvironmentType(e2eConfig.GetVariable(e2e.ManagementClusterEnvironmentVar))
+
+	switch infrastructureType {
+	case e2e.ManagementClusterEnvironmentKind:
+		chartMuseumInput.CustomIngressConfig = e2e.ChartMuseumIngress
```

Comment:
Is this ingress meant for mac users? Since chart museum is accessible locally and it didn’t need ingress